tests/tasks/setup_candlepin.yml

```diff
@@ -36,17 +36,8 @@
         use: "{{ __rhc_is_ostree |
                  ternary('ansible.posix.rhel_rpm_ostree', omit) }}"
 
-    - name: Stop and remove Candlepin container
-      command:
-        argv:
-          - podman
-          - stop
-          - candlepin
-      register: podman_stop_status
-      failed_when:
-        - podman_stop_status.rc != 0
-        - '"no such container" not in podman_stop_status.stderr'
-      changed_when: false
+    - name: Remove Candlepin container
+      include_tasks: remove_candlepin_container.yml
```

Comment:
According to my previous suggestions for the new task file for stopping candlepin, this new task file can be imported outside of this block, between the `Setup the test data` task and `Deploy Candlepin`